src/app/app-routing.module.ts

```diff
@@ -1,22 +1,22 @@
 // Based on the URL mapping in "routes" below, the RouterModule attaches
 // UI Components to the <router-outlet> element in the main AppComponent.
 
-import { NgModule }                  from '@angular/core';
-import { RouterModule, Routes }      from '@angular/router'
+import {NgModule} from '@angular/core';
+import {RouterModule, Routes} from '@angular/router'
 
-import { LoginComponent }            from './login.component'
-import { SelectRepositoryComponent } from './select-repository.component'
-import { CohortBuilderComponent }    from './cohort-builder.component'
+import {CohortBuilderComponent} from './cohort-builder.component'
```

Comment:
Nice I like this much better, not only because it requires less thinking (about aligning things) but also because it gives us more horizontal real estate.